src/main/java/seedu/address/logic/commands/FacebookConnectCommand.java

```diff
@@ -0,0 +1,99 @@
+package seedu.address.logic.commands;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import facebook4j.Facebook;
+import facebook4j.FacebookException;
+import facebook4j.FacebookFactory;
+import facebook4j.auth.AccessToken;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+
+/**
+ * Connects the addressbook to a personal Facebook account.
+ */
+public class FacebookConnectCommand extends Command {
+    public static final String COMMAND_WORD = "facebook connect";
+    public static final String COMMAND_ALIAS = "fbconnect";
+    public static final String MESSAGE_SUCCESS = "Connected to your Facebook Account!";
+
+    private static boolean authenticated = false;
+    private static String domain = "https://www.facebook.com/";
+    private static String appID = "131555220900267";
+    private static String commaSeparetedPermissions = "user_about_me,email,publish_actions,user_birthday,"
```

Comment:
Just some conventions, but since these values won't be changes you can use `public static final String NAME_IN_CAPS = ....`